src/Rings/groebner.jl

```diff
@@ -246,7 +248,7 @@ Ideal generating system with elements
 1 -> x*y - 3*x
 2 -> -2*x^2*y + y^3
 
-julia> B,m = Oscar.groebner_basis_with_transform(A, degrevlex(gens(R)))
+julia> B,m = Oscar.groebner_basis_with_transform(A, degrevlex(R))
```

Comment:
stupid question: You replaced gens(R) by R, because it is possible and more elegant, not because the other version is obsolete, right?

Reply:
Right.